docker/config.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+# Add host.docker.internal to /etc/hosts
+if ! grep -q "host.docker.internal" /etc/hosts; then
```

Comment:
Does this also need a note in the readme about the sudo privileges?

Reply:
done.